[2025-02-18T04:32:06.702+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:32:06.719+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-09-14T00:00:00+00:00 [queued]>
[2025-02-18T04:32:06.728+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-09-14T00:00:00+00:00 [queued]>
[2025-02-18T04:32:06.732+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:32:06.746+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-09-14 00:00:00+00:00
[2025-02-18T04:32:06.755+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=877) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:32:06.758+0000] {standard_task_runner.py:72} INFO - Started process 888 to run task
[2025-02-18T04:32:06.756+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-09-14T00:00:00+00:00', '--job-id', '213', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpy7i8kfdf']
[2025-02-18T04:32:06.760+0000] {standard_task_runner.py:105} INFO - Job 213: Subtask hello_task
[2025-02-18T04:32:06.775+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:06.817+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-09-14T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:32:06.819+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:06.872+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-14T00:00:00+00:00'
[2025-02-18T04:32:06.873+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:32:06.881+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:32:06.881+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:32:06.887+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:32:06.888+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-09-14T00:00:00+00:00, execution_date=20240914T000000, start_date=20250218T043206, end_date=20250218T043206
[2025-02-18T04:32:06.943+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:32:06.952+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:32:06.953+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
